GoogleDataTransport.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'GoogleDataTransport'
-  s.version          = '8.0.1'
+  s.version          = '8.2.0'
```

Comment:
Any reason for skipping 8.1.0? I believe it should be 8.1.0

Reply:
Yeah, I think it should be 8.1.0 (being in sync with the changelog as well).

Reply:
Sorry. I meant 8.1.0. Fixed.